aries-site/src/pages/templates/forms.js

```diff
@@ -29,11 +31,121 @@ const Forms = () => {
       />
       <ContentSection>
         <Subsection name={title} level={1} topic={topic}>
+          <SubsectionText>{page.description}</SubsectionText>
+          <Example
+            code="https://raw.githubusercontent.com/hpe-design/design-system/master/aries-site/src/examples/templates/forms/SimpleSignUpExample.js"
+            docs="https://v2.grommet.io/form?theme=hpe"
+            figma="https://www.figma.com/file/3fkwBelW5UsCbfwdDCJkT8/HPE-Form-Templates?node-id=1%3A2"
+          >
+            <SimpleSignUpExample />
+          </Example>
+        </Subsection>
+      </ContentSection>
+      <ContentSection>
+        <Subsection name="Guidance">
+          <SubsectionText>
+            Forms provide a way to collect important data from a user. However,
+            this data is only collected effectively when Forms are built in a
+            way in which users are inclined to fill out in full.
+          </SubsectionText>
+        </Subsection>
+        <Subsection name="Usage" level={3} gap="small">
           <SubsectionText>
-            Template patterns for common form use cases, from application
-            configurations to payment acceptance for the latest as-a-service
-            product offerings.
+            Forms don't need to be a tedious task for users to complete, as they
+            often come to be. It is important to ease as much of this effort as
+            possible when building a Form. Making Forms simple to complete leads
+            to increase in form completion.
           </SubsectionText>
+          <Text weight="bold">
+            Follow these guidelines when creating a Form:
+          </Text>
+          <BulletedList
+            items={[
+              'Use as few fields as possible',
+              'Wrap all input elements in a FormField',
+              'Fill in default values for as many fields as possible',
```

Comment:
I would also mention using pre filled inputs to reduce error. Like we have jane.smith@hpe.com in the signup example